wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt

```diff
@@ -167,7 +167,8 @@ internal constructor(
     val request = OkHttpUtils.buildStringGetRequest(endpointUrl, authToken)
 
     log.debug {
-      "Anchor account's all transactions request: assetCode = $assetCode, authToken = ${authToken.take(8)}"
+      "Anchor account's all transactions request: assetCode = $assetCode, authToken = ${authToken
+        .take(STRING_TRIM_LENGTH)}"
```

Comment:
I think we can remove "magic numbers" rule. I find it too strict, a lot of the time it's obvious from the context where it comes from